components/auth/signIn.tsx

```
@@ -1,139 +1,140 @@
-        </div>
-      </form>
-      <div className="mt-6 space-y-4">
-        <GoogleSignIn text="Sign In" />
-        <DiscordSignIn text="Sign In" />
-      </div>
-    </>
-  );
+				<div className="sm:flex sm:items-center sm:gap-4">
+					<button
+						className={`inline-block w-full sm:w-auto shrink-0 rounded-md border px-12 py-3 text-sm font-medium transition focus:outline-hidden focus:ring-3 text-black bg-accent hover:border-neon-green hover:bg-neon-green hover:bg-neon-green dark:hover:text-neon-green dark:border-neon-green dark:bg-neon-green dark:hover:bg-transparent`}
+					>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)